src/kaboom.ts

```diff
@@ -1080,7 +1010,39 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 	const game = {
 
 		// general events
-		ev: new EventHandler(),
+		ev: new EventHandler<{
+			mouseMove: [],
+			mouseDown: [MouseButton],
+			mousePress: [MouseButton],
+			mouseRelease: [MouseButton],
+			charInput: [string],
+			keyPress: [Key],
+			keyDown: [Key],
+			keyPressRepeat: [Key],
+			keyRelease: [Key],
+			touchStart: [Vec2, Touch],
+			touchMove: [Vec2, Touch],
+			touchEnd: [Vec2, Touch],
+			virtualButtonDown: [VirtualButton],
+			virtualButtonPress: [VirtualButton],
+			virtualButtonRelease: [VirtualButton],
+			gamepadButtonDown: [string],
+			gamepadButtonPress: [string],
+			gamepadButtonRelease: [string],
+			gamepadStick: [string, Vec2],
```

Comment:
Should these be typed to `GamepadButton` and `"left" | "right"` (I think we can also alias that to `GamepadStick`)?

Reply:
Sure sure